doc/how-to/getting_started_tcm.rst

```diff
@@ -0,0 +1,359 @@
+    Compiler: GNU-9.3.1
+    C_FLAGS: -fexceptions -funwind-tables -fasynchronous-unwind-tables -static-libstdc++ -fno-common -msse2  -fmacro-prefix-map=/home/centos/release/sdk/tarantool=. -std=c11 -Wall -Wextra -Wno-gnu-alignof-expression -fno-gnu89-inline -Wno-cast-function-type -O2 -g -DNDEBUG -ggdb -O2
+    CXX_FLAGS: -fexceptions -funwind-tables -fasynchronous-unwind-tables -static-libstdc++ -fno-common -msse2  -fmacro-prefix-map=/home/centos/release/sdk/tarantool=. -std=c++11 -Wall -Wextra -Wno-invalid-offsetof -Wno-gnu-alignof-expression -Wno-cast-function-type -O2 -g -DNDEBUG -ggdb -O2
+    $ tt version
+    Tarantool CLI EE 2.1.0, linux/amd64. commit: d80c2e3
+    $ tcm version
+    1.0.0-0-gd38b12c2
+
+Starting TCM
+------------
+
+|tcm_full_name| is ready to run out of the box. To start it, run the following command:
+
+..  code-block:: console
+
+    $ tcm --storage.etcd.embed.enabled
+
+The :ref:`--storage.etcd.embed.enabled <tcm_configuration_reference_storage_etcd_embed>`
+option makes |tcm| start its own instance of `etcd <https://etcd.io/>`__ on bootstrap.
+This ``etcd`` instance is used for storing the :ref:`TCM configuration <tcm_configuration>`.
+
+.. note::
+
+    During the development, it is also convenient to use the TCM-embedded etcd
+    as a configuration storage for Tarantool EE clusters connected to TCM it.
+    Learn more in :ref:`configuration_etcd`.
+
+Logging into TCM
+----------------
+
+#.  Open a web browser and go to ``http://127.0.0.1:8080/``.
+#.  Log in with the ``admin`` username. The initial password is generated automatically
+    and printed in the TCM log in a message like this:
+
+    ..  code-block:: text
+
+        Jan 24 05:51:28.443 WRN Generated super admin credentials login=admin password=qF3A5rjGurjAwmlYccJ7JrL5XqjbIHY6
+
+    Paste this password into the login form and click **Log in**.
+
+After a successful login, you see the |tcm| web UI:
+
+.. image:: images/tcm_start_empty_cluster.png
+    :width: 700
+    :align: center
+    :alt: TCM stateboard with empty cluster
+
+Setting up a Tarantool EE cluster
+---------------------------------
+
+To prepare a Tarantool EE cluster, complete the following steps:
+
+#.  Configure the cluster in |tcm|. |tcm| provides a web-based editor for writing
+    cluster configurations. It is connected to the configuration storage (etcd in
+    this case): all changes you make in the browser are sent to etcd in one click.
+#.  Start the cluster instances locally using the :ref:`tt <tt-cli>` utility.
+
+Configuring a cluster in TCM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A freshly installed |TCM| has a predefined cluster named **Default cluster**. It
+doesn't have any configuration or topology out of the box. Its initial properties
+include the default etcd and Tarantool connection parameters. Check these properties
+to find out where TCM sends the cluster configuration that you write.
+
+To view the **Default cluster**'s properties:
```

Comment:
Maybe, extract info about viewing configuration to a separate sub-section? Smth like this:

- Setting up a Tarantool EE cluster
   - Checking a cluster's connection settings
   - Configuring a cluster
   - Deploying the cluster locally